backend-project/small_eod/letters/views.py

```diff
@@ -12,3 +25,31 @@ class LetterViewSet(viewsets.ModelViewSet):
 class DescriptionViewSet(viewsets.ModelViewSet):
     queryset = Description.objects.all()
     serializer_class = DescriptionSerializer
+
+
+class FileViewSet(NestedViewSetMixin,
+                  mixins.CreateModelMixin,
+                  mixins.RetrieveModelMixin,
+                  viewsets.GenericViewSet):
+    model = File
+    serializer_class = FileSerializer
+    parent_lookup_kwargs = {
+        'letter_pk': 'letter__pk'
+    }
+
+    def perform_create(self, serializer):
+        serializer.save(letter=get_object_or_404(Letter, pk=self.kwargs['letter_pk']))
```

Comment:
Chyba w tym zakresie miałeś trudność. Zobacz na sekcje "[Passing additional attributes to .save()](https://www.django-rest-framework.org/api-guide/serializers/#passing-additional-attributes-to-save)".